cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java

```diff
@@ -36,49 +34,13 @@
         t3xMapping = new HashMap<>();
     }
 
-    public void add(String transformerId, String rtcId, PropertyBag rtc, PropertyBag ptc) {
-        if (t2xMapping.containsKey(transformerId)) {
-            throw new CgmesModelException("Transformer already added, Transformer id: " + transformerId);
-        }
-
-        CgmesRegulatingControlRatio rcRtc = null;
-        if (rtc != null) {
-            rcRtc = buildRegulatingControlRatio(rtcId, rtc);
-        }
-
-        CgmesRegulatingControlPhase rcPtc = null;
-        if (ptc != null) {
-            rcPtc = buildRegulatingControlPhase(ptc);
-        }
-
-        add(transformerId, rcRtc, rcPtc);
-    }
-
     public void add(String transformerId, CgmesRegulatingControlRatio rcRtc, CgmesRegulatingControlPhase rcPtc) {
         CgmesRegulatingControlForTwoWindingsTransformer rc = new CgmesRegulatingControlForTwoWindingsTransformer();
         rc.ratioTapChanger = rcRtc;
         rc.phaseTapChanger = rcPtc;
         t2xMapping.put(transformerId, rc);
     }
 
-    public void add(String transformerId, String rtcId2, PropertyBag rtc2, String rtcId3, PropertyBag rtc3) {
```

Comment:
Same question

Reply:
Is not used. It was used in the old conversion, deleted months ago.